packages/plugins/model-evaluate/tensorflow-bert-ner-model-evaluate/package.json

```
@@ -0,0 +1,39 @@
+{
+  "name": "@pipcook/plugins-tensorflow-bert-ner-model-evaluate",
+  "version": "1.0.4",
+  "description": "",
+  "main": "dist/index",
+  "types": "dist/index",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "npm run clean && npm run compile",
+    "clean": "rm -rf ./dist && rm -rf tsconfig.tsbuildinfo",
+    "compile": "tsc -b tsconfig.json"
+  },
+  "author": "Feely <feely@outlook.com>",
+  "license": "Apache 2.0",
+  "dependencies": {
+    "@pipcook/boa": "^1.0.4",
+    "@pipcook/pipcook-core": "^1.0.4"
+  },
+  "gitHead": "1e21ad0799e221dcf4b7aa3a3b2187d5b8d8b865",
+  "devDependencies": {
+    "@types/jasmine": "^3.5.7",
+    "nyc": "14.1.1"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "pipcook": {
+    "category": "modelTrain",
+    "datatype": "image"
```

This should be text?
```suggestion
    "datatype": "text"
```